### PR TITLE
[1599] fix apiv1 provider find campuses rendering

### PR DESCRIPTION
**Pull request by misaka**
### Context

The previous commit to add `mcb courses show` tweaked and broke rendering for `mcb apiv1 provider find`.

### Changes proposed in this pull request

Re-add campus table rendering.

### Checklist

- [ ] Make sure all information from the Trello card is in here
- [x] Attach to Trello card
- [ ] Rebased master
- [x] Cleaned commit history
- [x] Tested by running locally

### CODE REVIEW

**Comment by misaka** [comment]
[Fix "mcb apiv1 provider find" campuses table rendering](https://trello.com/c/m6QczJr9/1599-fix-mcb-apiv1-provider-find-campuses-table-rendering)

File: spec/factories/sites.rb [hunk 1]
```diff
@@ -24,7 +24,7 @@
     address3 { Faker::Address.city }
     address4 { Faker::Address.state }
     postcode { Faker::Address.postcode }
-    region_code { 'london' }
+    region_code { 1 }
```
Comment (benilovj) [inline spec/factories/sites.rb]: why do we need this change? `london` is more descriptive than `1`.

Comment (benilovj) [reply spec/factories/sites.rb]: Nothing seems to break if I put this change back, so I'm gonna amend the commit that introduced this change.

**Comment by benilovj** [comment]
I'm gonna merge this change because I would like to use the command that it fixes.

**Pull request merged**